src/App/Components/Common/NavProfile/Profile.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
why did you disable these rules?

Reply:
Those rules were raising issues because I was accessing the DOM and change states there with vanilla JS